docs/installation/sources/sources_linux.rst

```diff
@@ -0,0 +1,210 @@
+
+Python
+^^^^^^
+
+Download and install the Python_ version that better fits your requirements.
+
+CMake, g++, pip, wget and git
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can install CMake_, g++, pip_, wget_ and git_ using the package manager of your Linux distribution.
+For example, on Ubuntu you can install them with the command:
+
+.. code-block:: bash
+
+    sudo apt install cmake g++ pip wget git
+
+OpenSSL
+^^^^^^^
+
+You can install OpenSSL_ using the package manager of your Linux distribution.
+For example, on Ubuntu you can install it with the command:
+
+.. code-block:: bash
+
+   sudo apt install libssl-dev
+
+Gtest
+^^^^^
+
+.. note::
+
+    By default, *eProsima Fast DDS* does not compile tests.
+    You can activate them by adding the :code:`-DPERFORMANCE_TESTS=ON` flag when calling colcon_ or CMake_
+    (for details, see below).
+
+You can find information on how to install Gtest at this `link <https://github.com/google/googletest>`_.
+
+.. _colcon_installation_linux:
+
+Colcon Installation
+-------------------
+
+colcon_ is a command line tool to build sets of software packages.
+This section explains how to use it to compile easily *eProsima Fast DDS* and its dependencies.
+First, install the ROS2 development tools (colcon_ and vcstool_):
+
+.. code-block:: bash
+
+    pip install -U colcon-common-extensions vcstool
+
+.. note::
+
+    If this fails due to an Environment Error, add the :code:`--user` flag to your installation.
+
+Now, create a colcon_ workspace and download the repos file that will be used to install *eProsima Fast DDS* and
+its dependencies:
+
+.. code-block:: bash
+
+    $ mkdir Fast-DDS-ws && cd Fast-DDS-ws
+    $ wget https://raw.githubusercontent.com/eProsima/Fast-RTPS/master/fastrtps.repos
+    $ mkdir src
+    $ vcs import src < fastrtps.repos
+
+Finally, use colcon_ to compile all software:
+
+.. code-block:: bash
+
+    colcon build
+
+Once that’s finished building, you can source the new colcon overlay:
+
+.. code-block:: bash
+
+    source install/setup.bash
+
+.. important::
+
+    The sourcing of the local colcon overlay is required every time the colcon workspace is opened in a new shell
+    environment to run an *eProsima Fast DDS* instance.
+    As an alternative, you can add it permanently to you path by typing the following:
+
+    .. code-block:: bash
+
+        echo 'source PATH_TO_WORKSPACE/Fast-DDS-ws/install/setup.bash' >> ~/.bashrc
+
+    Where :code:`PATH_TO_WORKSPACE` is the path to the :code:`Fast-DDS-ws` worskspace.
+
+.. note::
+
+    If you want to compile the examples, you will need to add the flag
+    :code:`--cmake-args "-DCOMPILE_EXAMPLES=ON"` when running :code:`colcon build`.
+    If you want to compile the performance tests, you will need to add the flag
+    :code:`--cmake-args "--DPERFORMANCE_TESTS=ON"` when running :code:`colcon build`.
+    For this step, you need Gtest_ as explained in the requirements_linux_sources_ section above.
```

Comment:
In general, this will be covered in the cmake options section, we don't need it here

Reply:
ok. Deleted.